services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/util/CacheInvalidatingProcessor.java

```diff
@@ -17,6 +17,7 @@
 
 import static org.slf4j.LoggerFactory.getLogger;
 
+import edu.unc.lib.boxc.model.api.services.ContentPathFactory;
```

Comment:
Should this be with the other edu imports?

Reply:
I'm not sure why intellij put this here, guessing it didn't agree with eclipse about ordering. I've told it to organize imports